buggy-queries.txt

```diff
@@ -1,2 +0,0 @@
-what is the average salary of employees where name is not vinh
-how many jobs where name is not vinh
```

Comment:
the second case here doesn't work still I just removed it because it seemed like we didn't expect it to work. It still null pointers though, we need to fix that in the future!